general/1_terminal/marian-finiak/task3-finiak.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+read -p  'enter text: ' name
```

Comment:
Why?

Reply:
Sorry, I did not understand the task well, and I thought that we enter what day